feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/staking/start/common/selection/store/StartMultiStakingSelectionStore.kt

```diff
@@ -35,3 +39,21 @@ class RealStartMultiStakingSelectionStore : StartMultiStakingSelectionStore {
         }
     }
 }
+
+fun StartMultiStakingSelectionStore.getValidatorsOrEmpty(): List<Validator> {
+    val selection = currentSelection?.selection
+    if (selection is DirectStakingSelection) {
+        return selection.validators
+    }
+
+    return emptyList()
```

Comment:
return if (selection is DirectStakingSelection) {
selection.validators
} else {
emptyList()
}